coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
```

Comment:
You should remove this file from version control. You can add it to .gitignore, and you also need to use a variation of "git rm --cached coverage" to remove it from the existing repo